diff --git a/uhai/records/views.py b/uhai/records/views.py
index b611894..6cbd8c1 100644
--- a/uhai/records/views.py
+++ b/uhai/records/views.py
@@ -9,15 +9,7 @@ from forms import *
 from models import *
 from uhai.patients.models import Patient
 
-from uhai.core.views import model_view, role_model_view, user_model_view
-
-#CRUD
-order = login_required(lambda request, *args, **kwargs: role_model_view(request, *args, **kwargs))
-visit = login_required(lambda request, *args, **kwargs: role_model_view(request, *args, **kwargs))
-trackingfield = login_required(lambda request, *args, **kwargs: role_model_view(request, *args, **kwargs))
-encountertestresult = login_required(lambda request, *args, **kwargs: role_model_view(request, *args, **kwargs))
-encountertest = login_required(lambda request, *args, **kwargs: role_model_view(request, *args, **kwargs))
-problemtest = login_required(lambda request, *args, **kwargs: role_model_view(request, *args, **kwargs))
+from uhai.core.views import *
 
 @login_required
 def index(request, problem_type='', template_name = "records/index.html", *args, **kwargs):
